addons/mass_mailing/models/mailing.py

```diff
@@ -525,6 +527,51 @@ def _get_remaining_recipients(self):
         done_res_ids = [record['res_id'] for record in already_mailed]
         return [rid for rid in res_ids if rid not in done_res_ids]
 
+    def _send_statistics_mails(self):
```

Comment:
ChaussettesManagement: I would put a section (like Email Sending and TOOLS in the file) between those two, with STATISTICS naming. We can move this method in that section, so that everything is well organized.

Naming: I would name it _action_send_statistics (or something similar, at least _action and send keywords) to be coherent with guidelines.